src/coresyf_image_crop/examples.sh

```diff
@@ -1,4 +1,5 @@
 Aveiro_resampled.tif="http://example.org"
 #PASSING EXECUTION
 #Example of a passing execution
-./run --Ssource Aveiro_resampled.tif --Pbuffer 5 --Sgrid grid_EPSG_3763.zip --Ttarget myoutput.tif
\ No newline at end of file
+./run --Ssource tests/test_data/Aveiro_resampled.tif --Pbuffer 5 --Sgrid tests/test_data/grid_EPSG_3763 --Ttarget myoutput1.tif
+./run --Ssource tests/test_data/Aveiro_resampled.tif --Pbuffer 1000 --PbufferUnits meters --Sgrid tests/test_data/grid_EPSG_3763 --Ttarget myoutput2.tif
```

Comment:
This example line is in a wrong format. 
It should have two comment lines above it. The first is the test _title_ and the seconde the test _description_.
 ```bash
# Title
# Description
./run ...
```

Reply:
Done.